### PR TITLE
Switch to sigs.k8s.io/yaml from ghodss/yaml

### PR DESCRIPTION
**Pull request by dims**
Change-Id: Ic72b5131bf441d159012d67a6a3d87088d0e6d31

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
We now have a new repository named `sigs.k8s.io/yaml ` which has the code forked from `ghodss/yaml`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -26,12 +26,12 @@
 		},
 		{
 			"ImportPath": "cloud.google.com/go/compute/metadata",
-			"Comment": "v0.1.0-115-g3b1ae45",
+			"Comment": "v0.1.0-115-g3b1ae453",
```
Comment (cblecker) [inline Godeps/Godeps.json]: Can we remove these comment changes? (will hopefully be resolved once and for all soon)

**Review by cblecker — COMMENTED** [review]
rebase required
/approve

File: Godeps/Godeps.json (hunk 1 above)
Comment (dims) [reply Godeps/Godeps.json]: Done!

**Comment by cblecker** [comment]
/lgtm
/approve

🚀 🏆

**Comment by dims** [comment]
@smarterclayton @liggitt @thockin - can you please `/approve` (for `staging/src/k8s.io/apimachinery/pkg/apis/OWNERS`)

**Comment by neolit123** [comment]
https://cdn.shopify.com/s/files/1/0718/2803/t/3/assets/logo.png?5131102536933415569

**Pull request merged**